dali/python/nvidia/dali/triton.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import inspect
+
+
+def _discover_autoserialize(module, visited=[]):  # TODO test
+    """
+    TODO
+    :param module:
+    :return:
+    """
+    assert module is not None
+    ret = []
+    mems = inspect.getmembers(module)
+    modules = []
+    for mem in mems:
```

Comment:
I suggest a bit more verbose naming rather than `mem[0]` and `mem[1]`
```suggestion
    for name, member in module_members:
```

Reply:
Done